pipeline_steps/tempest.groovy

```diff
@@ -32,29 +32,27 @@ def tempest(){
   common.conditionalStage(
     stage_name: "Tempest Tests",
     stage: {
-      try{
-        def result = tempest_run()
-        def second_result = ""
-        if(result.contains("Race in testr accounting.")){
-          second_result = tempest_run()
-        }
-        if(second_result.contains("Race in testr accounting.")) {
-          currentBuild.result = 'FAILURE'
-        }
+      retry(2){
+        try{
+          def result = tempest_run()
+          if(result.contains("Race in testr accounting.")) {
+            currentBuild.result = 'FAILURE'
```

Comment:
I'd say yes.

What about the currentBuild.result? What is used for?
Should we remove that/modify that, so that it only get set when catch clause is reached?